day1/src/main.rs

```diff
@@ -0,0 +1,33 @@
+use std::env;
+use std::{
+    fs::File,
+    io::{BufRead, BufReader},
+};
+
+fn compute_fuel(x: i32) -> i32 {
+    if x > 0 {
+        ((x / 3) as i32) - 2
```

Comment:
Integer division will keep the same type and just floor any remainder, so this can be reduced to

```suggestion
        x / 3 - 2
```